DQM/SiStripMonitorClient/src/SiStripOfflineDQM.cc

```diff
@@ -68,12 +70,19 @@ SiStripOfflineDQM::SiStripOfflineDQM(edm::ParameterSet const& pSet) : configPar_
 
   usedWithEDMtoMEConverter_= configPar_.getUntrackedParameter<bool>("UsedWithEDMtoMEConverter",false); 
   createSummary_           = configPar_.getUntrackedParameter<bool>("CreateSummary",false);
+  createTkInfoFile_        = configPar_.getUntrackedParameter<bool>("CreateTkInfoFile",false);
   inputFileName_           = configPar_.getUntrackedParameter<std::string>("InputFileName","");
   outputFileName_          = configPar_.getUntrackedParameter<std::string>("OutputFileName","");
   globalStatusFilling_     = configPar_.getUntrackedParameter<int>("GlobalStatusFilling", 1);
   printFaultyModuleList_   = configPar_.getUntrackedParameter<bool>("PrintFaultyModuleList", false);
 
   nEvents_  = 0;
+
+  tkinfoTree_ = 0;
+  edm::Service<TFileService> fs;
```

Comment:
Just 82, something like `if (createTkInfoFile_){ edm::Service<TFileService> fs; tkinfoTree_ = fs->make<TTree>("TkDetIdInfo", ""); }`

For 81: pointers should never be assigned or compared to 0, nullptr should be used instead.